projects/novo-elements/src/elements/modal/ModalService.ts

```diff
@@ -1,20 +1,20 @@
 // NG2
-import { Injectable, ReflectiveInjector } from '@angular/core';
+import { Injectable, ViewContainerRef, StaticProvider, Type } from '@angular/core';
 // APP
 import { NovoModalRef, NovoModalParams, NovoModalContainerElement } from './Modal';
 import { ComponentUtils } from './../../utils/component-utils/ComponentUtils';
 
 @Injectable()
 export class NovoModalService {
-  _parentViewContainer: any = null;
+  _parentViewContainer: ViewContainerRef;
 
   constructor(private componentUtils: ComponentUtils) {}
 
-  set parentViewContainer(view) {
+  set parentViewContainer(view: ViewContainerRef) {
     this._parentViewContainer = view;
   }
 
-  open(component, scope = {}) {
+  open<T>(component: Type<T>, scope = {}) {
```

Comment:
Should the generic type argument be defaulted to any to preserve backward compatibility?